docs/pages/settings.md

```diff
@@ -1,6 +1,20 @@
+---
+hide:
+  - navigation
+---
+
+<style>
+.md-content .md-typeset h1 { 
+  position: absolute;
+  left: -999px;
+}
+</style>
+
+
 # Settings
 
 Here are the environment variables that can be used to configure the Aaana SDK:
+
 - TMP_DATA_DIR: The directory to store temporary data. Default: `/tmp/aana`.
 - NUM_WORKERS: The number of request workers. Default: `2`.
 - DB_CONFIG: The database configuration in the format `{"datastore_type": "sqlite", "datastore_config": {"path": "/path/to/sqlite.db"}}`. Currently only SQLite and PostgreSQL are supported. Default: `{"datastore_type": "sqlite", "datastore_config": {"path": "/var/lib/aana_data"}}`.
```

Comment:
If the user wants to a add it via cmd or via .env file, it's better to remove the spacing from the env var
Not critical
`DB_CONFIG='{"datastore_type":"sqlite","datastore_config":{"path":"/path/to/db"}}'`

Reply:
It should with with spaces as well, no? I would prefer to keep spaces for better readability.